test_communication/CMakeLists.txt

```diff
@@ -65,21 +65,27 @@ if(BUILD_TESTING)
     if(with_message_argument)
       # adding test for each message type
       foreach(message_file ${message_files})
+        set(_SKIP_TEST "")
```

Comment:
It was to avoid confusion with the value stored in it that is also `SKIP_TEST` but the leading underscore is misleading indeed, I'll find a better name